README.md

```diff
@@ -1,2 +1,54 @@
-# ballerinax-cdata-connectors
-Manage Ballerina CData connector modules centrally.
+Ballerina CData Connect Cloud Connector
+===================
+
+[![Build](https://github.com/ballerina-platform/module-ballerinax-cdata.connect/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-cdata.connect/actions?query=workflow%3ACI)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-cdata.connect.svg)](https://github.com/ballerina-platform/module-ballerinax-cdata.connect/commits/master)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+[CData Connect Cloud](https://cloud.cdata.com) is a consolidated connectivity platform that links applications to a broad ecosystem of real-time data sources through consistent, standards-compliant interfaces. CData Connect Cloud provides tools to seamlessly access data from any system, anywhere.
+
+The CData Connect Cloud [Ballerina](https://ballerina.io/) connector allows you to programmatically access all of the CData Connect Cloud applications, databases, APIs, services via the Java Database Connectivity (JDBC) API. 
+It provides operations to execute a wide range of standard DDL Commands, SQL Commands, and SQL Functions for querying data sources. 
+You can find reference information for all the CData Connect Cloud SQL commands (DDL, DML, and query syntax) [here](https://cloud.cdata.com/docs/SQL-Reference.html).
+
+For more information, go to the module(s).
+- [cdata.connect](Module.md)
+
+## Building from the source
+
+### Setting up the prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 11. You can install either [OpenJDK](https://adoptopenjdk.net/) or [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html).
+
+    > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+
+2. Download and install [Ballerina Swan Lake](https://ballerina.io/). 
+
+### Building the source
+
+Execute the commands below to build from the source.
+
+- To build the library:
+    ```shell
+    ./gradlew clean build
+    ```
+- To run the integration tests: 
+    ```shell
+    ./gradlew clean test
+    ```
+
+## Contributing to Ballerina
+
+As an open source project, Ballerina welcomes contributions from the community. 
+
+For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/gsheet/CONTRIBUTING.md).
```

Comment:
Is this a valid link?

Reply:
Resolved in [b4c6902](https://github.com/ballerina-platform/module-ballerinax-cdata.connect/pull/3/commits/b4c6902204c43dea5ebd37f46f4dbcb023738216). Thank you.